contrib/xbacklight_linux.lua

```diff
@@ -0,0 +1,26 @@
+local tonumber = tonumber
+local io = { popen = io.popen }
+local setmetatable = setmetatable
+
+-- xbacklight: provides backlight level
+-- vicious.contrib.xbacklight
+local xbacklight = {}
+
+local function xbacklight_cmd(args)
+    local f = io.popen("xbacklight "..args)
+    if f == nil then
+      return nil
+    else
+      local line = f:read("*all")
+      f:close()
+      return line
```

Comment:
Please avoid `io.popen`. In this case, you can use our local library `spawn.easy_async` which wraps Awesome `awful.spawn`. Please look for other widget types that uses this function for examples of how this can be done.